### PR TITLE
Support new alternative Intel map URLs

### PR DESCRIPTION
**Pull request by Blaisorblade**
Ensure this plugin loads when IITC loads, for instance when loading from `ingress.com/intel` or Mission permalinks (never tested the second part, I followed the code).

This follows the IITC updates in these commits:
iitc-project/ingress-intel-total-conversion@32e455f4e8beba65d9072566dddf20ca4133efb6
iitc-project/ingress-intel-total-conversion@eba7d388a0af1163f46221edaba3aeab77b6d7f8

Caveat: double-test for typos, I've edited the script locally with TamperMonkey, tested it minimally, and pasted the changes here via GitHub. And I didn't double-check the change much, beyond digging down the original IITC changes.

As explained [on the community](https://plus.google.com/105383756361375410867/posts/WPm9tu6ZY7C)
for the first fix (not for the mission links):

> Why does IITC/[third-party plugin X] not load in Chrome?
> tl;dr: Quick fix: ensure you visit www.ingress.com/intel instead of ingress.com/intel.
> ingress.com no longer redirects to www.ingress.com, so the URL matching patterns
> broke. Third-party plugins must update their @include/@match patterns to be
> "*.ingress.com" instead of just "www.ingress.com" to fix this.

### CODE REVIEW

**Comment by pmodin** [comment]
Why include `/mission/` URLs?

**Comment by Blaisorblade** [comment]
That follows IITC itself. Their change was motivated as "Enable IITC on mission permalink", as described in commit iitc-project/ingress-intel-total-conversion@eba7d38 (already linked above).

Since those permalinks lead to a full Intel window (I just tried now), it makes sense to enable all plugins, even Maxfields.

**Comment by itayo** [comment]
Im sorry for being slow right now.  Il take a proper look within the next
24 hours
